src/hud/debug_hud.rs

```diff
@@ -19,15 +19,17 @@ pub struct DebugHud {
 
     coordinates_last: Point3<f32>,
     coordinates_geometry_buffers: GeometryBuffers<u16>,
+
+    ratio: f32,
 }
 
 impl DebugHud {
     pub fn new(render_context: &RenderContext) -> Self {
         let text_renderer = TextRenderer::new(render_context).unwrap();
         let fps_geometry_buffers =
-            text_renderer.string_to_buffers(&render_context, -0.98, 0.97, "");
+            text_renderer.string_to_buffers(&render_context, -0.98, 0.97, 16.0/9.0, "");
         let coordinates_geometry_buffers =
-            text_renderer.string_to_buffers(&render_context, -0.98, 0.97 - text_renderer::DY, "");
+            text_renderer.string_to_buffers(&render_context, -0.98, 0.97 - text_renderer::DY, 16.0/9.0, "");
```

Comment:
Can't suggest changes on this somehow, but maybe

```rust
let ratio = 16.0 / 9.0;
let fps_geometry_buffers =
    text_renderer.string_to_buffers(&render_context, -0.98, 0.97, ratio, "");
let coordinates_geometry_buffers =
    text_renderer.string_to_buffers(&render_context, -0.98, 0.97 - text_renderer::DY, ratio, "");
```

Then using that `ratio` for the `DebugHud` construction down below